pages/library/[libGameId]/index.jsx

```diff
@@ -18,9 +46,30 @@ export default function LibraryGameDetails() {
 
   if (error) return <div>Error: {error.message}</div>;
 
+  function handleAddGameFromLib(newGame) {
+    if (games?.some((game) => Number(game.id) === newGame[0].id)) {
```

Comment:
Once you store the object from the array in a separate variable, you can write it like this:

```suggestion
    if (games?.some((game) => Number(game.id) === newGame.id)) {
```